{"name": "test_get_booking", "status": "broken", "statusDetails": {"message": "AttributeError: 'dict' object has no attribute 'status_code'", "trace": "self = <tests.test.Integration.test_case_2.TestPostDeleteVerify object at 0x0000018623B4CB60>\ncreate_booking = 3223\n\n    def test_get_booking(self,create_booking):\n        booking_id=create_booking\n        get_url=APIConstants.put_patch_delete_url(self,booking_id=booking_id)\n        response=get_request(url=get_url,auth=None)\n>       verify_http_response_code(response=response,expected_data=404)\n\ntest_case_2.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse = {'additionalneeds': 'Breakfast', 'bookingdates': {'checkin': '2018-01-01', 'checkout': '2019-01-01'}, 'depositpaid': True, 'firstname': 'Sumit', ...}\nexpected_data = 404\n\n    def verify_http_response_code(response,expected_data):\n>       assert response.status_code==expected_data,\"Failed-ER!=AR\"\nE       AttributeError: 'dict' object has no attribute 'status_code'\n\n..\\..\\..\\src\\helpers\\common_verifications.py:3: AttributeError"}, "start": 1739293019659, "stop": 1739293021907, "uuid": "256415d7-e0df-48ea-88b5-c70aafec7c5d", "historyId": "498c5d30637783891892cf44929a9cb9", "testCaseId": "498c5d30637783891892cf44929a9cb9", "fullName": "tests.test.Integration.test_case_2.TestPostDeleteVerify#test_get_booking", "labels": [{"name": "parentSuite", "value": "tests.test.Integration"}, {"name": "suite", "value": "test_case_2"}, {"name": "subSuite", "value": "TestPostDeleteVerify"}, {"name": "host", "value": "someshcom065"}, {"name": "thread", "value": "5132-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.Integration.test_case_2"}]}